src/utils/auth.utils.ts

```diff
@@ -0,0 +1,29 @@
+import bcrypt from 'bcrypt';
+import { v4 as uuidv4 } from 'uuid';
+
+/**
+ * Hashes a password using bcrypt.
+ * @param password - The password to hash.
+ * @returns The hashed password.
+ */
+export const hashPassword = async (password: string): Promise<string> => {
+  return await bcrypt.hash(password, 10);
```

Comment:
move this cost factor to config.
or it is better that cost factor should be random always.
whats your opinion